core/arch/arm/plat-imx/conf.mk

```diff
@@ -219,7 +251,9 @@ CFG_DT ?= y
 CFG_PAGEABLE_ADDR ?= 0
 CFG_PSCI_ARM32 ?= y
 CFG_SECURE_TIME_SOURCE_REE ?= y
+CFG_CSU ?= y
 CFG_UART_BASE ?= UART1_BASE
+CFG_IMX_CAAM ?= y
```

Comment:
This is a bit funny.
You turn the CAAM off for everybody and then turn it back on.
Shouldn't it be on for everybody and then just switched off for the 6ull ?

Reply:
Hi @bryanodonoghue 
Do I turn off the CAAM for everybody?

What I do for CFG_IMX_CAAM is:
* Disable the CAAM if the platform requires it. (What I do for 6ull)
* Enable the CAAM by default if CFG_IMX_CAAM not already set before.

Would you do it differently?

Reply:
Yes I see viewing the whole file.
Looks fine.

Reviewed-by: `Bryan O'Donoghue <bryan.odonoghue@linaro.org>`

Reply:
Tags applied.